src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,355 @@ java -D&lt;property&gt;=&lt;value&gt;  \
       </div>
     </section>
 
+        <section name="Examples">
+        <h3>1. Using a Configuration File (-c)</h3>
+        <p><strong>Goal:</strong> Run Checkstyle using a specified configuration file.</p>
+
+        <p><strong>Configuration File (config.xml):</strong></p>
+     <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
+    &lt;?xml version="1.0"?&gt;
+    &lt;!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "https://checkstyle.org/dtds/configuration_1_3.dtd"&gt;
+    &lt;module name="Checker"&gt;
+        &lt;module name="TreeWalker"&gt;
+            &lt;module name="FallThrough"/&gt;
+        &lt;/module&gt;
+    &lt;/module&gt;
+    </code></pre></div>
+
+        <p><strong>Test File (Test.java):</strong></p>
+    <div class="wrapper"><pre class="prettyprint"><code class="language-java">
+    class Test {
+        public void foo() {
```

Comment:
Can we make indentation start from 0 , to remove extra indentation in final html rendering?